app/src/main/java/com/depromeet/team6/presentation/ui/main/MainViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package com.depromeet.team6.presentation.ui.main
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.depromeet.team6.data.datalocal.manager.LockServiceManager
+import com.depromeet.team6.domain.repository.UserInfoRepository
+import com.google.firebase.messaging.FirebaseMessaging
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import javax.inject.Inject
+
+@HiltViewModel
+class MainViewModel @Inject constructor(
+    private val userInfoRepository: UserInfoRepository,
+    private val lockServiceManager: LockServiceManager
+) : ViewModel() {
+
+    private val _fcmToken = MutableLiveData<String?>()
+    val fcmToken: LiveData<String?> = _fcmToken
```

Comment:
뷰모델 안에서만 사용하는 변수인거 같은데 LiveData 로 선언한 이유가 뭔가요?

Reply:
```suggestion
       private var fcmToken: String? = null
```
MainActivity에 위치하는게 부자연스러워서 급하게 뷰모델로 이동하려다 보니 지피티를 돌려서 그런건데 말씀해주신 것 처럼 뷰모델 내부에서만 사용하기 때문에 위 suggestion 처럼 변경할 생각입니다! 
괜찮을까요